rest-servlet/src/main/java/org/hawkular/inventory/rest/RequestUtil.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.inventory.rest;
+
+import org.hawkular.inventory.api.paging.Order;
+import org.hawkular.inventory.api.paging.PageContext;
+import org.hawkular.inventory.api.paging.Pager;
+
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.UriInfo;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Lukas Krejci
+ * @since 0.0.1
+ */
+public class RequestUtil {
+
+    private RequestUtil() {
+    }
+
+    public static Pager extractPaging(UriInfo uri) {
+        MultivaluedMap<String, String> params = uri.getQueryParameters();
+
+        String pageS = params.getFirst("page");
+        String perPageS = params.getFirst("per_page");
```

Comment:
While I saw that from the other guy. Is that `per_page` really common? Or should we consider `pageSize` or `page_size`?